cmd/hc/hc.go

```diff
@@ -88,8 +88,8 @@ func setupApp() (app *cli.App) {
 				},
 			},
 			Action: func(c *cli.Context) error {
-				srcPath := c.Args().First()
-				if srcPath == "" {
+				cloneSrcPath := c.Args().First()
+				if cloneSrcPath == "" {
```

Comment:
From scratch, I'd probably use 'clone' or clonePath here, I'm fine with whatever you choose and will update, even if that is to revert.